packages/project-disaster-trail/src/components/LandingPage/index.js

```diff
@@ -4,10 +4,9 @@ import { Link } from "react-router";
 import styled from "styled-components";
```

Comment:
Can we import `styled` from emotion instead of styled components? Trying to remove styled components as a dependency. Not urgent.

Reply:
Also could use emotion css prop, instead if preferred.

Reply:
You betcha.